src/api/db/migrate/20160824132643_fix_bs_request_counter.rb

```diff
@@ -0,0 +1,10 @@
+class FixBsRequestCounter < ActiveRecord::Migration
+  def self.change
+    # BsRequestCounter is not set correctly
+    # Introduced with 20160321105300_request_counter.rb
+    # See https://github.com/openSUSE/open-build-service/issues/2068
+    counter = BsRequest.reorder(:number).pluck(:number).last || 0
+    BsRequestCounter.destroy_all
+    BsRequestCounter.create!(counter: counter + 1)
```

Comment:
I see, this is needed for the next commit. In that case please squash both commits, since they belong together.